CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
 * [#3140](https://github.com/bbatsov/rubocop/pull/3140): `Style/FrozenStringLiteralComment` works with file doesn't have any tokens. ([@pocke][])
 * [#3154](https://github.com/bbatsov/rubocop/issues/3154): Fix handling of `()` in `Style/RedundantParentheses`. ([@lumeet][])
 * [#3160](https://github.com/bbatsov/rubocop/pull/3160): `Style/Lambda` fix whitespacing when auto-correcting unparenthesized arguments. ([@palkan][])
+* [#3175](https://github.com/bbatsov/rubocop/pull/3175): Generate 'Exclude' list for style cops to `.rubocop_todo.yml` if different styles is used. ([@flexoid][])
```

Comment:
if different styles **are** used

And it's not entirely correct to say that it's for style cops, because there are a handful of rails cops that have a configurable enforced style, and they will also be affected by this correction. So say "cops with configurable enforced style" instead.